File: runtime/CSlib/key.cs
namespace bigloo 
{
  public sealed class key: cnst 
  {
    public static readonly key _key= new key( 0x106 );

    private key( int  n )
      : base( n ) 
    {
    }

    public override void write( output_port  p ) 
    {
      p.write( "#!key" );
    }
  }
}

File: runtime/Jlib/key.java
package bigloo;

public final class key extends cnst
{
  public static final key key= new key( 0x106 );

  private key( final int  n )
  {
    super( n );
  }

  public void write( final output_port  p )
  {
    p.write( "#!key" );
  }
}
